<testsuites>
	<testsuite id="1" tests="1" failures="1" package="C:\Yerel_Disk_D\Karaca\KaracaMobilProject\specs\AndroidSpecs\run.spec" time="0.773" timestamp="2023-02-01T12:03:05" name="MobilProject Scenarios" errors="0" hostname="DESKTOP-TC8IL10">
		<properties></properties>
		<testcase classname="MobilProject Scenarios" name="OrnekSenaryoKaraca" time="0.773">
			<failure message="OrnekSenaryoKaraca&#xA;Scenario Pre Hook Failure: &#39;org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: The application at &#39;driver/driver.apk&#39; does not exist or is not accessible&#xA;Build info: version: &#39;3.141.59&#39;, revision: &#39;e82be7d358&#39;, time: &#39;2018-11-14T08:17:03&#39;&#xA;System info: host: &#39;DESKTOP-TC8IL10&#39;, ip: &#39;192.168.1.4&#39;, os.name: &#39;Windows 10&#39;, os.arch: &#39;amd64&#39;, os.version: &#39;10.0&#39;, java.version: &#39;1.8.0_202&#39;&#xA;Driver info: driver.version: AndroidDriver&#xA;remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: The application at &#39;driver/driver.apk&#39; does not exist or is not accessible&#xA;    at getResponseForW3CError (C:\Users\testinium\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)&#xA;    at asyncHandler (C:\Users\testinium\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:380:37)&#xA;Build info: version: &#39;3.141.59&#39;, revision: &#39;e82be7d358&#39;, time: &#39;2018-11-14T08:17:03&#39;&#xA;System info: host: &#39;DESKTOP-TC8IL10&#39;, ip: &#39;192.168.1.4&#39;, os.name: &#39;Windows 10&#39;, os.arch: &#39;amd64&#39;, os.version: &#39;10.0&#39;, java.version: &#39;1.8.0_202&#39;&#xA;Driver info: driver.version: AndroidDriver&#39;" type="OrnekSenaryoKaraca&#xA;Scenario Pre Hook Failure: &#39;org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: The application at &#39;driver/driver.apk&#39; does not exist or is not accessible&#xA;Build info: version: &#39;3.141.59&#39;, revision: &#39;e82be7d358&#39;, time: &#39;2018-11-14T08:17:03&#39;&#xA;System info: host: &#39;DESKTOP-TC8IL10&#39;, ip: &#39;192.168.1.4&#39;, os.name: &#39;Windows 10&#39;, os.arch: &#39;amd64&#39;, os.version: &#39;10.0&#39;, java.version: &#39;1.8.0_202&#39;&#xA;Driver info: driver.version: AndroidDriver&#xA;remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: The application at &#39;driver/driver.apk&#39; does not exist or is not accessible&#xA;    at getResponseForW3CError (C:\Users\testinium\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)&#xA;    at asyncHandler (C:\Users\testinium\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:380:37)&#xA;Build info: version: &#39;3.141.59&#39;, revision: &#39;e82be7d358&#39;, time: &#39;2018-11-14T08:17:03&#39;&#xA;System info: host: &#39;DESKTOP-TC8IL10&#39;, ip: &#39;192.168.1.4&#39;, os.name: &#39;Windows 10&#39;, os.arch: &#39;amd64&#39;, os.version: &#39;10.0&#39;, java.version: &#39;1.8.0_202&#39;&#xA;Driver info: driver.version: AndroidDriver&#39;">org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: The application at &#39;driver/driver.apk&#39; does not exist or is not accessible&#xA;Build info: version: &#39;3.141.59&#39;, revision: &#39;e82be7d358&#39;, time: &#39;2018-11-14T08:17:03&#39;&#xA;System info: host: &#39;DESKTOP-TC8IL10&#39;, ip: &#39;192.168.1.4&#39;, os.name: &#39;Windows 10&#39;, os.arch: &#39;amd64&#39;, os.version: &#39;10.0&#39;, java.version: &#39;1.8.0_202&#39;&#xA;Driver info: driver.version: AndroidDriver&#xA;remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: The application at &#39;driver/driver.apk&#39; does not exist or is not accessible&#xA;    at getResponseForW3CError (C:\Users\testinium\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)&#xA;    at asyncHandler (C:\Users\testinium\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:380:37)&#xA;Build info: version: &#39;3.141.59&#39;, revision: &#39;e82be7d358&#39;, time: &#39;2018-11-14T08:17:03&#39;&#xA;System info: host: &#39;DESKTOP-TC8IL10&#39;, ip: &#39;192.168.1.4&#39;, os.name: &#39;Windows 10&#39;, os.arch: &#39;amd64&#39;, os.version: &#39;10.0&#39;, java.version: &#39;1.8.0_202&#39;&#xA;Driver info: driver.version: AndroidDriver&#xD;&#xA;&#x9;at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:208)&#xD;&#xA;&#x9;at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:217)&#xD;&#xA;&#x9;at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:239)&#xD;&#xA;&#x9;at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)&#xD;&#xA;&#x9;at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)&#xD;&#xA;&#x9;at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)&#xD;&#xA;&#x9;at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)&#xD;&#xA;&#x9;at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)&#xD;&#xA;&#x9;at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:131)&#xD;&#xA;&#x9;at io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:38)&#xD;&#xA;&#x9;at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:84)&#xD;&#xA;&#x9;at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:94)&#xD;&#xA;&#x9;at io.appium.java_client.android.AndroidDriver.&lt;init&gt;(AndroidDriver.java:95)&#xD;&#xA;&#x9;at com.testinium.HookImpl.beforeScenario(HookImpl.java:53)&#xD;&#xA;&#x9;at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#xD;&#xA;&#x9;at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#xD;&#xA;&#x9;at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#xD;&#xA;&#x9;at java.lang.reflect.Method.invoke(Method.java:498)&#xD;&#xA;&#x9;at com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:30)&#xD;&#xA;&#x9;at com.thoughtworks.gauge.execution.HooksExecutor$TaggedHookExecutor.executeHook(HooksExecutor.java:90)&#xD;&#xA;&#x9;at com.thoughtworks.gauge.execution.HooksExecutor$TaggedHookExecutor.execute(HooksExecutor.java:78)&#xD;&#xA;&#x9;at com.thoughtworks.gauge.execution.HooksExecutor.execute(HooksExecutor.java:35)&#xD;&#xA;&#x9;at com.thoughtworks.gauge.processor.MethodExecutionMessageProcessor.executeHooks(MethodExecutionMessageProcessor.java:55)&#xD;&#xA;&#x9;at com.thoughtworks.gauge.processor.ScenarioExecutionStartingProcessor.process(ScenarioExecutionStartingProcessor.java:24)&#xD;&#xA;&#x9;at com.thoughtworks.gauge.RunnerServiceHandler.lambda$startScenarioExecution$5(RunnerServiceHandler.java:132)&#xD;&#xA;&#x9;at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)&#xD;&#xA;&#x9;at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)&#xD;&#xA;&#x9;at java.lang.Thread.run(Thread.java:748)&#xD;&#xA;Caused by: java.lang.reflect.InvocationTargetException&#xD;&#xA;&#x9;at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#xD;&#xA;&#x9;at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#xD;&#xA;&#x9;at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#xD;&#xA;&#x9;at java.lang.reflect.Method.invoke(Method.java:498)&#xD;&#xA;&#x9;at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:186)&#xD;&#xA;&#x9;... 27 more&#xD;&#xA;Caused by: org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: The application at &#39;driver/driver.apk&#39; does not exist or is not accessible&#xA;Build info: version: &#39;3.141.59&#39;, revision: &#39;e82be7d358&#39;, time: &#39;2018-11-14T08:17:03&#39;&#xA;System info: host: &#39;DESKTOP-TC8IL10&#39;, ip: &#39;192.168.1.4&#39;, os.name: &#39;Windows 10&#39;, os.arch: &#39;amd64&#39;, os.version: &#39;10.0&#39;, java.version: &#39;1.8.0_202&#39;&#xA;Driver info: driver.version: AndroidDriver&#xA;remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: The application at &#39;driver/driver.apk&#39; does not exist or is not accessible&#xA;    at getResponseForW3CError (C:\Users\testinium\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)&#xA;    at asyncHandler (C:\Users\testinium\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:380:37)&#xD;&#xA;&#x9;at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#xD;&#xA;&#x9;at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)&#xD;&#xA;&#x9;at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)&#xD;&#xA;&#x9;at java.lang.reflect.Constructor.newInstance(Constructor.java:423)&#xD;&#xA;&#x9;at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)&#xD;&#xA;&#x9;at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)&#xD;&#xA;&#x9;at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)&#xD;&#xA;&#x9;at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)&#xD;&#xA;&#x9;at java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)&#xD;&#xA;&#x9;at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126)&#xD;&#xA;&#x9;at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:498)&#xD;&#xA;&#x9;at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:485)&#xD;&#xA;&#x9;at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)&#xD;&#xA;&#x9;at java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:152)&#xD;&#xA;&#x9;at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)&#xD;&#xA;&#x9;at java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:464)&#xD;&#xA;&#x9;at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)&#xD;&#xA;&#x9;... 32 more&#xD;&#xA;</failure>
		</testcase>
		<system-out></system-out>
		<system-err></system-err>
	</testsuite>
</testsuites>